mhcnames/allele_name.py

```diff
@@ -131,7 +131,13 @@ def parse_allele_name(name, species_prefix=None):
 
     sep, name = parse_separator(name)
 
-    allele_code, name = parse_numbers(name)
+    allele_code, rest_of_text = parse_numbers(name)
```

Comment:
So the variable `name` on the left was incorrectly named previously?

If so, LGTM!